x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -184,7 +184,11 @@
                                 // Kibana user will read / write to these indices
                                 RoleDescriptor.IndicesPrivileges.builder()
                                     .indices(ReservedRolesStore.ALERTS_INDEX)
-                                    .privileges("all").build()
+                                    .privileges("all").build(),
+                                // Endpoint / Fleet policy responses. Kibana requires read access to send telemetry
+                                RoleDescriptor.IndicesPrivileges.builder()
+                                    .indices(".ds-metrics-endpoint.policy-*")
```

Comment:
1) "Endpoint / Fleet policy responses" Do such responses contain user data (some time of metrics are OK)? We usually avoid granting Kibana access to user data. Please acknowledge this.

2) When granting privileges we consider the backing index of a data stream a low level abstraction. Whenever defining roles, it is enough to use the data stream name, eg: 
```suggestion
                                    .indices("metrics-endpoint.policy-*")
```
This is more true in user facing definitions; I'm fine with it like this since this is "hidden".

Reply:
> "Endpoint / Fleet policy responses" Do such responses contain user data (some type of metrics are OK)? We usually avoid granting Kibana access to user data. Please acknowledge this.

That's correct - it's being accessed for telemetry usage. These are fleet policy _responses_ that **do not contain** any PII or sensitive information. They are being accessed to check for failed policy configurations from the context of endpoint installs. This will enable us to improve onboarding/UX and find bugs on our end.

> When granting privileges we consider the backing index of a data stream a low level abstraction. Whenever defining roles it is enough to use the data stream name, ...

Good to know. Thanks!